### DISCUSSION TITLE
Fixing a Bug in a To-Do List Application

### DISCUSSION THREAD

**Discussion by SuperHeroDev** [community/community#110202] [Programming Help] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

I'm developing a simple To-Do List application, but there seems to be a bug where tasks are not being deleted properly. When you click the delete button next to a task, nothing happens.
Here's the relevant part of HTML and JavaScript code:
HTML:
`<ul id="taskList">
    <li><input type="checkbox" onchange="toggleTaskCompletion(this)">Task 1<button onclick="deleteTask(this)">Delete</button></li>
    <li><input type="checkbox" onchange="toggleTaskCompletion(this)">Task 2<button onclick="deleteTask(this)">Delete</button></li>
</ul>
`
JavaScript:
`function deleteTask(button) {
    var task = button.parentNode;
    task.parentNode.removeChild(task);
}
`

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by ghost** [comment, marked as answer]
Check your browser console for better debugging...and here

JavaScript:
```js
function deleteTask(button) { var task = button.parentNode; task.parentNode.removeChild(task); } 
```

I guess you are just deleting the parent node from the parent that is deleting 'task' from task...

**Comment by ettaboyle** [comment]
Thanks for posting in the GitHub Community, @SuperHeroDev! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the **GitHub products Issues and Projects**. I've gone ahead and moved it for you. Good luck!
